showcase/src/app/app.component.html

```diff
@@ -54,9 +54,8 @@
 					</button>
 				</div>
 			</div>
-
-			<div class="stark-app-bar-page-title">
-				<h1>STARK</h1>
+			<div class="stark-header-extra">
```

Comment:
Could you please put back the title of page here, and replace the placeholder for logout at lines 32-34 by `<stark-app-logout></stark-app-logout>` component ?